docs/book/src/user/quick-start.md

```diff
@@ -702,7 +702,7 @@ Calico is used here as an example.
 
 ```bash
 kubectl --kubeconfig=./capi-quickstart.kubeconfig \
-  apply -f https://docs.projectcalico.org/v3.15/manifests/calico.yaml
+  apply -f https://docs.projectcalico.org/v3.18/manifests/calico.yaml
```

Comment:
Not sure about that one. We're using kindnet in our CAPD e2e tests. I tested it manually and it worked, but if we want to stay on 3.15 I can revert it again.

Reply:
I don't see problem in using a more recent release.